internal/controller/runtime/fsm/runtime_fsm_waiting_for_shoot_reconcile.go

```diff
@@ -49,7 +49,7 @@ func sFnWaitForShootReconcile(_ context.Context, m *fsm, s *systemState) (stateF
 			imv1.ConditionTypeRuntimeProvisioned,
 			imv1.ConditionReasonAuditLogConfigured,
```

Comment:
The used condition `imv1.ConditionReasonAuditLogConfigured` should also be adapted to changes. I suppose `imv.ConditionReasonProcessingCompleted` is the correct value here, but please confirm it.